parser/src/main/scala/aqua/parser/expr/ConstantExpr.scala

```diff
@@ -0,0 +1,25 @@
+package aqua.parser.expr
+
+import aqua.parser.Expr
+import aqua.parser.lexer.Token._
+import aqua.parser.lexer.{Name, Value}
+import aqua.parser.lift.LiftParser
+import cats.Comonad
+import cats.parse.{Parser => P}
+
+case class ConstantExpr[F[_]](
+  name: Name[F],
+  value: Value[F],
+  mark: Boolean
+) extends Expr[F]
+
+object ConstantExpr extends Expr.Leaf {
+
+  override def p[F[_]: LiftParser: Comonad]: P[ConstantExpr[F]] = {
+    (((`const` *> ` ` *> Name
+      .p[F] <* ` `) ~ `?`.? <* `=` <* ` `).with1 ~ Value.`value`).map {
```

Comment:
Probably `with1` should be at `?`.?, it converts the whole expression to `Parser0` currently.